### ISSUE TITLE
RadioButtonGroup doesn't follow a11y best practice

### ISSUE THREAD

**Issue by knoobie** [vaadin/vaadin-radio-button#131]
Take a look at 

* https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20161214/examples/radio/radio-1/radio-1.html 
* http://a11y-style-guide.com/style-guide/section-forms.html

Wrong usage of aria-*, missing attributes and wrong html elements.

**Comment by anezthes** [comment]
Chrome and VoiceOver:
- All radio buttons are announced as "1 of 1".

Safari and VoiceOver:
- Name of group is never announced.

**Comment by juuso-vaadin** [comment]
Required groups are not announced.

**Comment by web-padawan** [comment]
Closing in favor of https://github.com/vaadin/web-components/issues/94 which describes the problem in more detail.
